managed/resource_cluster.go

```diff
@@ -82,6 +82,24 @@ and modify the backup schedule of the cluster being created.`,
 						Type:     types.StringType,
 						Required: true,
 					},
+					"num_cores": {
+						Description: "Number of CPU cores in the nodes of the region.",
+						Type:        types.Int64Type,
+						Optional:    true,
+						Computed:    true,
+					},
+					"disk_size_gb": {
+						Description: "Disk size of the nodes of the region.",
+						Type:        types.Int64Type,
+						Optional:    true,
+						Computed:    true,
+					},
+					"disk_iops": {
+						Description: "Disk IOPS of the nodes of the region.",
+						Type:        types.Int64Type,
+						Optional:    true,
+						Computed:    true,
+					},
```

Comment:
What happens if these parameters are set asymmetrically and TF is used without the feature being enabled in prod? Does it fail gracefully?

Reply:
If the parameters are passed before the feature is enabled in apiserver prod, the request will fail in apiserver validation. Please let me know if I understand your question incorrectly.